pytrip/cube.py

```diff
@@ -431,8 +431,9 @@ def create_dicom_base(self):
         meta.TransferSyntaxUID = UID.ImplicitVRLittleEndian  # Implicit VR Little Endian - Default Transfer Syntax
         ds = FileDataset("file", {}, file_meta=meta, preamble=b"\0" * 128)
         ds.PatientsName = self.patient_name
-        if self.patient_id is None:
-            ds.PatientID = datetime.datetime.today().strftime('%Y%m%d-%H%M%S')
+        if self.patient_id is None or self.patient_id == '':
+            self.patient_id = datetime.datetime.today().strftime('%Y%m%d-%H%M%S')
+        ds.PatientID = self.patient_id
```

Comment:
alternatively one could create self.patient_id in cube.__init__.
It is mandatory for dicoms only, but need to be in sync across trip files. 
Creating it at cube.__init__ level can be overridden still, e.g. when importing from dicom etc.

So, either method is OK with me.